src/st/dre/1E1C8.c

```diff
@@ -365,8 +365,65 @@ void func_8019F304(void) {
     }
 }
 
-// DECOMP_ME_WIP func_8019F3BC https://decomp.me/scratch/Hfk9n
-INCLUDE_ASM("st/dre/nonmatchings/1E1C8", func_8019F3BC);
+void func_8019F3BC(Entity* arg0) {
+    s16 temp_v1;
+    s32 temp_v1_2;
+    s32 var_a0;
+    s32 var_v1;
+    s32 temp_s0;
+
+    if (arg0->step == 0) {
+        InitializeEntity(g_InitializeEntityData0);
+        arg0->animSet = 2;
+        arg0->palette = 0x81B6;
+        arg0->unk6C = 0x70;
+        arg0->zPriority = 0xC0;
+        arg0->drawFlags |= 0xC;
+        arg0->drawMode |= 0x30;
+
+        switch (arg0->ext.generic.unk84.U8.unk0) {
```

Comment:
I just saw https://github.com/Xeeynamo/sotn-decomp/issues/744 to not use `ext.generic.*` any more. @Xeeynamo is there a way to tell which entity type is most likely? The calls to this function haven't been mapped yet so may need to wait.